### ISSUE TITLE
Toasts are not announced with VoiceOver on Chrome

### ISSUE THREAD

**Issue by robinmetral** [sumup-oss/circuit-ui#1706]
## Describe the bug

In my testing with VoiceOver on Chrome, toasts are not announced when appended to the live region.

## Steps to reproduce

1. Use Chrome and VoiceOver on macOS
2. Go to [the story in full-screen mode](https://circuit.sumup.com/iframe.html?args=&id=notification-notificationtoast--base&viewMode=story) (otherwise, the Storybook iframe causes issues with live regions) 
2. Turn on VoiceOver
4. Click the button to open a toast
5. Notice that the toast is not announced

## Expected behavior

Since the toast is appended to a live region with `role="status"`, its contents should be announced.

## Specifications

* Version: latest (5.2.3)
* Browser: Chrome 104.0.5112.79
* Screen reader: VoiceOver
* OS: macOS

## Additional context

The toast is announced as expected using VoiceOver on Firefox and Safari.

In [this trimmed down example](https://codesandbox.io/s/testing-live-regions-g089d2?file=/index.html), new content in the live region is correctly announced on Chrome. I wasn't able to reproduce the issue outside of the Circuit UI Storybook yet.

**Comment by robinmetral** [comment]
Since our markup seems correct in this instance, since the error doesn't occur on Firefox, and since this can't be reproduced in CodeSandbox, I'll treat this as a bug in Storybook+Chrome+VO and close this for now. If we get similar reports from production we can investigate it further.
